tests/unit/training/test_training_sae.py

```diff
@@ -4,7 +4,11 @@
 import torch
 
 from sae_lens.sae import SAE
-from sae_lens.training.training_sae import TrainingSAE, TrainingSAEConfig
+from sae_lens.training.training_sae import (
+    TrainingSAE,
+    TrainingSAEConfig,
+    _calculate_topk_aux_acts,
```

Comment:
If this function is intended to be imported by other files/classes, probably should not be an underscore/internal function.

Reply:
It's only imported by its own unit test, it should not be imported externally which is why it has an underscore